src/cryptoadvance/specter/util/psbt.py

```diff
@@ -76,6 +76,8 @@ def vout_to_dict(self, vout: TransactionOutput) -> dict:
         return obj
 
     def to_dict(self) -> dict:
+        if not self.tx:
+            return
```

Comment:
Isn't it guaranteed that it should return a dict? So why not:
```suggestion
            return {}
```

Reply:
Yes, I think your way is best.